src/main/kotlin/lotto/Lotto.kt

```diff
@@ -0,0 +1,27 @@
+package lotto
+
+class Lotto {
```

Comment:
```suggestion
class Lotto(
    val value: List<Int>
 {
```
Lotto는 반드시 여섯 개의 숫자 리스트를 가지고 있습니다.
그렇다면 캡슐화를 해보면 어떨까요?

Reply:
캡슐화를 진행하니 코드가 많이 수정되었네요. 캡슐화 진행하였습니다~!